sdk/tools/azure-openrewrite/src/main/resources/META-INF/rewrite/migrationRecipes/core/com.azure.core.http.policy.yml

```diff
@@ -0,0 +1,267 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------
+# Migration Recipes for azure.core.http.policy
+# --------------------------------------------------------------------
+type: specs.openrewrite.org/v1beta/recipe
+name: com.azure.openrewrite.recipe.azure.core.http.policy
+displayName: Migrate from azure.core.http.policy to next generation stack
+description: This recipe migrates azure.core.http.policy package to the next generation stack.
+recipeList:
+
+  # --------------------------------------------------------------------
+  # AddDatePolicy
+  # --------------------------------------------------------------------
+
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: com.azure.core.http.policy.AddDatePolicy
+      newFullyQualifiedTypeName: com.azure.v2.core.http.policy.AddDatePolicy
```

Comment:
The imports in this recipe are pointing to the wrong package name. All these policies are in clientcore.